components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift

```diff
@@ -45,4 +51,118 @@ open class HeaderStackViewTypicalUseSwiftExample: HeaderStackViewTypicalUse {
+                        compatibleWith: nil)
+    let imageView = UIImageView(image: image)
+    imageView.frame = topView.bounds
+    imageView.contentMode = .scaleAspectFill
+    imageView.autoresizingMask = [ .flexibleWidth, .flexibleHeight ]
+    topView.clipsToBounds = true
+    topView.addSubview(imageView)
+
+    navBar.titleTextAttributes = itemTitleTextAttributes()
+
+    let moreButton = UIBarButtonItem(title: "Reveal",
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(didTapToggleButton))
+    moreButton.setTitleTextAttributes(itemTitleTextAttributes(), for: .normal)
+    navBar.trailingBarButtonItems = [ moreButton ]
+  }
+
+  func didTapToggleButton() {
+    toggled = !toggled
+    UIView.animate(withDuration: 0.4, animations: {
+      var frame = self.stackView.frame
+      if (self.toggled) {
+        frame.size.height = 200
+      } else {
+        frame.size = self.stackView.sizeThatFits(self.stackView.bounds.size)
+      }
+      self.stackView.frame = frame
+      self.stackView.layoutIfNeeded()
+    })
+  }
+
+  func itemTitleTextAttributes() -> [String: Any] {
+    return [ NSForegroundColorAttributeName : UIColor.white ]
+  }
+}
+
+class ExampleInstructionsViewHeaderStackViewSwiftTypicalUse : UIView {
+  override func draw(_ rect: CGRect) {
+    UIColor.white.setFill()
+    UIBezierPath(rect: rect).fill()
+    let textSizeRect = textSize(forRect: rect)
+    let rectForText = CGRect(x: rect.origin.x + rect.size.width / 2 - textSizeRect.width / 2,
+                             y: rect.origin.y + rect.size.height / 2 - textSizeRect.height / 2,
+                             width: textSizeRect.width,
+                             height: textSizeRect.height)
+    instructionsString().draw(in: rectForText)
+    drawArrow(withFrame: CGRect(x: rect.size.width / 2 - 12,
+                                y: rect.size.height / 2 - 58 - 12,
+                                width: 24,
+                                height: 24))
+  }
+
+  func textSize(forRect rect: CGRect) -> CGSize {
+    return instructionsString()
+      .boundingRect(with: rect.size,
+                    options: NSStringDrawingOptions.usesLineFragmentOrigin,
+                    context: nil).size
+  }
+
+  func instructionsString() -> NSAttributedString {
+    let style = NSMutableParagraphStyle()
+    style.alignment = .center
+    style.lineBreakMode = .byWordWrapping
+    let instructionsDictionary1 = [
+      NSFontAttributeName : UIFont.preferredFont(forTextStyle: .headline),
+      NSForegroundColorAttributeName : MDCPalette.grey.tint600.withAlphaComponent(0.87),
+      NSParagraphStyleAttributeName : style
+    ]
+
+    let instructionsDictionary2 = [
+      NSFontAttributeName : UIFont.preferredFont(forTextStyle: .subheadline),
+      NSForegroundColorAttributeName : MDCPalette.grey.tint600.withAlphaComponent(0.87),
```

Comment:
We have a color scheme, was it unused in the previous example code?

Reply:
Yeah it was unused.